ci/job-variables.yml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+variables:
```

Comment:
There seem to be a lot of other shared variables, can we throw those in here as well?

Reply:
You mean outside of the variables added by this PR? Yes, I guess we could add those here as well. I'll look into that in a separate PR. Feels a bit out of scope for this one.

Reply:
I was thinking of things like `    - name: git_branch_sha
      value: $[ dependencies.git_sha.outputs['out.branch'] ]` which seem new in a few places

Reply:
Oh, I see. Yes, moving these into the template seems to work.